frontend/src/locales/scripts/en.json5

```diff
@@ -175,7 +175,7 @@
     },
     heading: {
       image: "Image Sources",
-      audio: "Audio Sources",
+      audio: "Audio track Sources",
```

Comment:
Please revert this change as "track" is not needed here. A source of audio tracks can be called an "audio source".